commit 04c4854fdfbc11b2b6a2cf1c8045f2efcd5dcdcc
Author: Xinchen Hui <laruence@gmail.com>
Date:   Fri Aug 17 13:35:15 2018 +0800

    Fixed bug #76754 (parent private constant in extends class memory leak)

diff --git a/Zend/tests/bug76754.phpt b/Zend/tests/bug76754.phpt
new file mode 100644
index 0000000..424f1fb
--- /dev/null
+++ b/Zend/tests/bug76754.phpt
@@ -0,0 +1,18 @@
+--TEST--
+Bug #76754 (parent private constant in extends class memory leak)
+--INI--
+opcache.enable=0
+opcache.enable_cli=0
+--FILE--
+<?php
+
+class FOO
+{
+	private const FOO = 'BAR';
+}
+
+class BAR extends FOO { }
+?>
+okey
+--EXPECT--
+okey
